crates/ruff_python_semantic/src/binding.rs

```diff
@@ -25,7 +25,8 @@ pub struct Binding<'a> {
     pub scope: ScopeId,
     /// The context in which the [`Binding`] was created.
     pub context: ExecutionContext,
-    /// The statement in which the [`Binding`] was defined.
+    /// The statement in which the [`Binding`] was defined. `None` if the [`Binding`]
+    /// comes from a built-in.
```

Comment:
Please double-check the accuracy of these comments. In any case, I'd like to document the cases so we don't have to run through this uncertainty in the future.